getUserFeatureFlag/main.go

```diff
@@ -15,7 +14,7 @@ import (
 	"github.com/aws/aws-sdk-go/service/dynamodb/dynamodbattribute"
 )
 
-func processGetById(userId string, flagId string) (*models.FeatureFlagUserMapping, error) {
+func processGetById(userId string, flagId string) (*utils.FeatureFlagUserMappingResponse, error) {
 
 	db := database.CreateDynamoDB()
 
```

Comment:
can you comment near line no 24 that `userId`  is the partition key and `flagId` is the sort key

Reply:
Done